MapboxNavigation/MGLStyle.swift

```diff
@@ -25,55 +25,31 @@ extension MGLStyle {
      */
     public class var navigationNightStyleURL: URL {
         get {
-            if(MGLAccountManager.hasChinaBaseURL){
-                return mapboxChinaDayStyleURL
+            if MGLAccountManager.hasChinaBaseURL {
+                return mapboxChinaNightStyleURL
```

Comment:
Not sure why `mapboxChinaDayStyleURL` was used in `navigationNightStyleURL`, probably was used by mistake.